src/agent/src/config.rs

```diff
@@ -22,6 +22,7 @@ const DEFAULT_CONTAINER_PIPE_SIZE: i32 = 0;
 const VSOCK_ADDR: &str = "vsock://-1";
 const VSOCK_PORT: u16 = 1024;
 const SERVER_ADDR_ENV_VAR: &str = "KATA_AGENT_SERVER_ADDR";
+const LOG_LEVEL_ENV_VAR: &str = "KATA_LOG_LEVEL";
```

Comment:
@jodh-intel, firstly, thanks for the patches!

One question, would make sense to just add a comment explicitly mentioning that the env-var shoud only be used for testing purposes?

Reply:
also, I wonder if this variable should start with `KATA_AGENT_`

Reply:
Sure - done.

Reply:
@devimc - I did think about that but decided against it. But now you've convinced me it should have an agent prefix - branch updated ;)